msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/results/ResetPasswordResult.kt

```diff
@@ -120,23 +77,13 @@ sealed interface ResetPasswordSubmitCodeResult : Result {
     class PasswordRequired(
         override val nextState: ResetPasswordPasswordRequiredState
     ) : ResetPasswordSubmitCodeResult, Result.SuccessResult(nextState = nextState)
-
-    /**
-     * CodeIncorrect ErrorResult, which indicates the verification code provided by user is incorrect.
-     * The code should be re-submitted.
-     *
-     * @param error [com.microsoft.identity.nativeauth.statemachine.IncorrectCodeError]
-     */
-    class CodeIncorrect(
-        override val error: IncorrectCodeError
-    ) : ResetPasswordSubmitCodeResult, Result.ErrorResult(error = error)
 }
 
 /**
  * Sign in resend code result, produced by
  * [com.microsoft.identity.nativeauth.statemachine.states.ResetPasswordCodeRequiredState.resendCode]
```

Comment:
All of these references need to be updated to point to the new package (this should have been part of Saurabh's package name refactoring, but we overlooked it)

Reply:
This is updated with the new package. Does this comment apply for some other line?

Reply:
Yes. E.g. the comment "SSPR start result". @Yuki-YuXin please double check that it's updated everywhere.